developer/js/source/lexical-model-compiler/script-overrides-decorator.ts

```diff
@@ -0,0 +1,73 @@
+/// <reference path="../../node_modules/@keymanapp/models-types/index.d.ts" />
+
+export function decorateWithScriptOverrides(breaker: WordBreakingFunction, option: OverrideScriptDefaults) {
+  if (option !== 'break-words-at-spaces') {
+    throw new Error(`Unsupported script override: ${option}`)
+  }
+
+  /**
+   * Matches a string that contains only characters that are letters or  according
+   * to Unicode. (e.g.,  their General Category is Lu, Ll, Lt, Lm, Lo, Mn, Mc, or Me).
+   *
+   * Last updated for unidata 12.1.0.
```

Comment:
We currently have Unicode 13.0 data in our repo at
resources/standards-data/unicode-character-database/

Are you able to make use of UnicodeData.txt? I see it has the General Category info

Reply:
Will do! And yes, I'll see if I can use `resources/standards-data/unicode-character-database/UnicodeData.txt` in this code 👍